benchmark/cmd/benchmark.go

```diff
@@ -114,6 +114,24 @@ func compare(jobComparisonData *util.JobComparisonData) {
 	}
 }
 
+// Pretty print results of the comparison.
+func printResults(jobComparisonData *util.JobComparisonData) {
+	glog.Infof("Metric-wise results of comparison for 50th percentile:")
+	jobComparisonData.PrettyPrintWithFilter(func(k util.MetricKey, d util.MetricComparisonData) bool {
+		return k.Percentile != "Perc50"
+	})
+	glog.Infof("")
+	glog.Infof("Metric-wise results of comparison for 90th percentile:")
+	jobComparisonData.PrettyPrintWithFilter(func(k util.MetricKey, d util.MetricComparisonData) bool {
+		return k.Percentile != "Perc90"
+	})
+	glog.Infof("")
+	glog.Infof("Metric-wise results of comparison for 99th percentile:")
+	jobComparisonData.PrettyPrintWithFilter(func(k util.MetricKey, d util.MetricComparisonData) bool {
```

Comment:
I think the order should be reversed - 99th percentile is more important, hence it should be on top.